dbms/tests/queries/0_stateless/01013_hex_float.sql

```diff
@@ -0,0 +1,7 @@
+SELECT hex(1.0);
+SELECT hex(101.);
+SELECT hex(1e+18);
+SELECT hex(1e-20);
+SELECT hex(1e+100);
+SELECT hex(0.000578);
+SELECt hex(-123.987);
```

Comment:
Need to add test cases for Float32.

Reply:
Need to add a test case with non-constant value:
by SELECT-ing from `numbers` table function.

(look at the examples in tests around)